$OpenBSD: patch-configure,v 1.3 2008/12/29 17:03:46 sturm Exp $

OpenBSD uses /var/db/gconf which is detected correctly by other code.

--- configure.orig	Sun Dec 14 04:41:47 2008
+++ configure	Sat Dec 20 09:08:43 2008
@@ -4232,12 +4232,6 @@ fi
 
 
 
-# Override gconf default install locations.  The default is to always
-# put them into /etc.  This change puts them into $(prefix)/etc.
-if test "x$GCONF_SCHEMA_INSTALL_SOURCE" = "x"; then
-  GCONF_SCHEMA_CONFIG_SOURCE="xml::${sysconfdir}/gconf/gconf.xml.defaults"
-fi
-GCONF_SCHEMA_CONFIG_SOURCE_DIRONLY=`echo $GCONF_SCHEMA_CONFIG_SOURCE | cut -d ':' -f 3`
 
 
 
